core/modules/user/views/user.views.inc

```diff
@@ -102,7 +102,7 @@ function user_views_data() {
   $data['users']['uid_current'] = array(
     'real field' => 'uid',
     'title' => t('Current'),
-    'help' => t('Filter the view to the currently logged in user.'),
+    'help' => t('Filter the view to the account of the currently logged in person.'),
```

Comment:
though they can't do the logging in themselves, they are *technically* what becomes logged in!  People aren't actually crawling around inside your website ;)